.github/workflows/test.yml

```diff
@@ -1,43 +1,63 @@
 name: Test
 
-on: [push]
-
+on:
+  schedule:
+    - cron: "0 0 9 * *"
```

Comment:
Interesting! Once a month on day 9 if I'm reading this right.

Reply:
Oops that wasn't right